src/coinbase/address.ts

```diff
@@ -253,19 +272,19 @@ export class Address {
    * Creates a trade model for the specified amount and assets.
    *
    * @param amount - The amount of the Asset to send.
-   * @param fromAssetId - The ID of the Asset to trade from. For Ether, eth, gwei, and wei are supported.
-   * @param toAssetId - The ID of the Asset to trade to. For Ether, eth, gwei, and wei are supported.
+   * @param fromAsset - The Asset to trade from. For Ether, eth, gwei, and wei are supported.
```

Comment:
remove ` For Ether, eth, gwei, and wei are supported.`

Reply:
good catch! updated.